src/main/java/com/chopshop166/chopshoplib/maps/DifferentialDriveMap.java

```diff
@@ -0,0 +1,22 @@
+package com.chopshop166.chopshoplib.maps;
+
+import com.chopshop166.chopshoplib.outputs.SendableSpeedController;
+
+import edu.wpi.first.wpilibj.GyroBase;
+
+/**
+ * Differential Drive Map
+ * <p>
+ * Gets the Left {@link SendableSpeedController}, gets the right
+ * {@link SendableSpeedController}, and gets {@link Gyro}
```

Comment:
This should reference `GyroBase` not `Gyro` (which doesn't exist.

Reply:
Gyro does exist and I had it at first, I'll change it tonight

Reply:
Ah. It is an interface in wpilib. Unfortunately I don't believe you can link to things outside of chopshoplib as the documentation builder is unaware of them (I assume). I think you'll have to remove the link entirely.

Reply:
Instead of removing the link, just mention that it's a GyroBase instead. That's what it actually returns. More accurately, though, you could say something like "a gyro representing the angle and direction of the robot" or similar

You can link to external things by specifying the full package of them, or importing them.